static/truewiki/truewiki.css

```diff
@@ -506,10 +532,23 @@ header {
 }
 
 #search {
-    position: absolute;
-    right: 30px;
+    position: -webkit-sticky; /* Safari */
```

Comment:
That comment feels a bit weird :P I would just leave it out .. everyone who does anything with CSS knows to look it up first if he wants to remove it :D